<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverlayConfig</class>
 <widget class="QWidget" name="OverlayConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="qswOverlayPage">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="qwOverlayConfig">
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="qgbOptions">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Options</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QCheckBox" name="qcbEnable">
               <property name="toolTip">
                <string>Enable overlay.</string>
               </property>
               <property name="whatsThis">
                <string>This sets whether the overlay is enabled or not. This setting is only checked when applications are started, so make sure Mumble is running and this option is on before you start the application.&lt;br /&gt;Please note that if you start Mumble after starting the application, or if you disable the overlay while the application is running, there is no safe way to restart the overlay without also restarting the application.</string>
               </property>
               <property name="text">
                <string>Enable Overlay</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item>
                <widget class="QPushButton" name="qpbLoadPreset">
                 <property name="toolTip">
                  <string>Load an overlay preset from file</string>
                 </property>
                 <property name="text">
                  <string>Load…</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="qpbSavePreset">
                 <property name="toolTip">
                  <string>Save your overlay settings to file</string>
                 </property>
                 <property name="text">
                  <string>Save…</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="qpbUninstall">
                 <property name="text">
                  <string>Uninstall Overlay</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="qgpFps">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>FPS and Clock Display</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item>
                <widget class="QCheckBox" name="qcbShowFps">
                 <property name="toolTip">
                  <string>Display a frame counter in the overlay</string>
                 </property>
                 <property name="text">
                  <string>Show FPS counter</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="qcbShowTime">
                 <property name="toolTip">
                  <string>Display a clock in the overlay showing the current local time (system time).</string>
                 </property>
                 <property name="text">
                  <string>Show Clock</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="qpbFpsFont">
                 <property name="toolTip">
                  <string>Set the overlay font.</string>
                 </property>
                 <property name="whatsThis">
                  <string/>
                 </property>
                 <property name="text">
                  <string>Font</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="qpbFpsColor">
                 <property name="toolTip">
                  <string>Set the overlay text color.</string>
                 </property>
                 <property name="whatsThis">
                  <string/>
                 </property>
                 <property name="text">
                  <string>Color</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGraphicsView" name="qgvFpsPreview">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>256</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="backgroundBrush">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>128</red>
                  <green>128</green>
                  <blue>128</blue>
                 </color>
                </brush>
               </property>
               <property name="foregroundBrush">
                <brush brushstyle="NoBrush">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </property>
               <property name="renderHints">
                <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="qtwSetup">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="qwLayoutTab">
          <attribute name="title">
           <string>Layout</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QGraphicsView" name="qgvView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="backgroundBrush">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
              </brush>
             </property>
             <property name="renderHints">
              <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="qwExceptions">
          <attribute name="title">
           <string>Overlay Exceptions</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Overlay Exception Mode</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="qcbOverlayExclusionMode"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QWidget" name="qwLaunchers" native="true">
               <layout class="QVBoxLayout" name="qvblLaunchers">
                <item>
                 <widget class="QLabel" name="qlLaunchersLabel">
                  <property name="text">
                   <string>Allowed launchers</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PathListWidget" name="qlwLaunchers" native="true"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QPushButton" name="qpbLaunchersAdd">
                    <property name="text">
                     <string>Add</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="qpbLaunchersRemove">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Remove</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="qwWhitelist" native="true">
               <layout class="QVBoxLayout" name="qvblWhitelist">
                <item>
                 <widget class="QLabel" name="qlWhitelistLabel">
                  <property name="text">
                   <string>Allowed programs</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PathListWidget" name="qlwWhitelist" native="true"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QPushButton" name="qpbWhitelistAdd">
                    <property name="text">
                     <string>Add</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="qpbWhitelistRemove">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Remove</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="qwPaths" native="true">
               <layout class="QVBoxLayout" name="qvblPaths">
                <item>
                 <widget class="QLabel" name="qlPathsLabel">
                  <property name="text">
                   <string>Allowed paths</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PathListWidget" name="qlwPaths" native="true"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QPushButton" name="qpbPathsAdd">
                    <property name="text">
                     <string>Add</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="qpbPathsRemove">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Remove</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="qwBlacklist" native="true">
               <layout class="QVBoxLayout" name="qvblBlacklist">
                <item>
                 <widget class="QLabel" name="qlvBlacklistLabel">
                  <property name="text">
                   <string>Blacklisted programs</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PathListWidget" name="qlwBlacklist" native="true"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QPushButton" name="qpbBlacklistAdd">
                    <property name="text">
                     <string>Add</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="qpbBlacklistRemove">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Remove</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="qwOverlayInstall">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Overlay Installation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="qlInstallIcon">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="mumble.qrc">:/config_osd.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="qlInstallText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Mumble has detected that you do not have the Mumble Overlay installed.

Click the button below to install the overlay.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="qpbInstall">
              <property name="text">
               <string>Install Mumble Overlay</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="qwOverlayUpgrade">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Overlay Upgrade</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="qlUpgradeIcon">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="mumble.qrc">:/config_osd.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="qlUpgradeText">
              <property name="text">
               <string>Mumble has detected an old version of the overlay support files installed on your computer.

To upgrade these files to their latest versions, click the button below.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="qpbUpgrade">
              <property name="text">
               <string>Upgrade Mumble Overlay</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PathListWidget</class>
   <extends>QListWidget</extends>
   <header>PathListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mumble.qrc"/>
 </resources>
 <connections/>
</ui>
